src/TestSuite/TestCase.php

```diff
@@ -38,17 +38,16 @@
 use Closure;
 use Exception;
 use LogicException;
-use Mockery;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
 use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase as BaseTestCase;
 use ReflectionClass;
 use ReflectionException;
 use function Cake\Core\pluginSplit;
 
 /**
  * Cake TestCase class
  */
-abstract class TestCase extends BaseTestCase
+abstract class TestCase extends MockeryTestCase
```

Comment:
We can choose whether to extend `MockeryTestCase` (which in turn extends `PHPUnit\Framework\TestCase`). Or whether to use the traits `MockeryPHPUnitIntegration` and `MockeryTestCaseSetUp`

Reply:
I am in favour of traits.